code/game/machinery/telecomms/broadcasting.dm

```diff
@@ -176,6 +176,9 @@
 				if(independent_radio.independent && independent_radio.can_receive(frequency, signal_reaches_every_z_level))
 					radios += independent_radio
 
+	for(var/obj/item/radio/called_radio in radios)
```

Comment:
```suggestion
	for(var/obj/item/radio/called_radio as anything in radios)
```

Reply:
wait what does the as everything do here?

Reply:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STANDARDS.md#avoid-unnecessary-type-checks-and-obscuring-nulls-in-lists